.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2.1
```

Comment:
We would prefer a github workflow here.
I don't see a problem using github here. We even wouldn't need a container as we only need Perl::Tidy. We can just use the default ubuntu runner and just install the right version.
I suggest to grep the version out of `cpanfile`, because that might be the file we all have in common.

Reply:
I was more focused on CircleCI because it was not apparent how could I run an arbitrary docker image from GH Actions.

But as of latest commit I like a lot the result: 

![image](https://github.com/os-autoinst/os-autoinst-common/assets/5963207/446086e7-1d12-4a46-9a26-6636abc9e2ff)